runtime/doc/os_win32.txt

```diff
@@ -175,7 +176,27 @@ There was a special version of gvim that runs under Windows 3.1 and 3.11.
 Support was removed in patch 7.4.1364.
 
 ==============================================================================
-7. Win32 mini FAQ					*win32-faq*
+7. Installation package					*win32-installer*
+
+A simple installer for windows is available at http://www.vim.org/download.php
+(stable version) and nightly builds are also available at
+https://github.com/vim/vim-win32-installer/releases/
+
+The nightly builds include 32bit and 64bit builds, have most features enabled
+and usually also contain an extra cryptographic signed installer, so Windows
+will not complain.
+
+To use the installer, simply run the exe file.  The following switches are
+also supported: >
+
+    gvim_<version>.exe /S          -> silent install without any dialogues
+    gvim_<version>.exe /D=C:\vim   -> Install into directory c:\vim
```

Comment:
`/D=` must be the last argument. So, `/S /D=C:\vim` is accepted, but `/D=C:\vim /S` is not accepted.
Moreover, `/D=` can be used without `/S`, but it might not so useful.
So, how about something like this?
```suggestion
    gvim_<version>.exe /S            -> silent install without any dialogues
    gvim_<version>.exe /S /D=C:\vim  -> Install into directory c:\vim silently
```

The other part looks good to me.